node/pkg/aggregator/aggregator.go

```diff
@@ -102,16 +114,14 @@ func (n *Aggregator) HandleTriggerMessage(ctx context.Context, msg raft.Message)
 		return err
 	}
 	defer n.leaveOnlyLast10Entries(triggerMessage.RoundID)
-
+	n.mu.Lock()
+	defer n.mu.Unlock()
```

Comment:
the scope which n.mu.Lock() and n.roundTriggers.mu.Lock() are having are different so it probably shouldn't have the same result 😅 

to be more precise, it might be better to have separate mutex for `roundLocalAggregate` value